bundles/org.openhab.binding.nuvo/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,758 @@
+				<label>Zone 16</label>
+				<description>The Controls for Zone 16</description>
+			</channel-group>
+			<channel-group id="zone17" typeId="zone17">
+				<label>Zone 17</label>
+				<description>The Controls for Zone 17</description>
+			</channel-group>
+			<channel-group id="zone18" typeId="zone18">
+				<label>Zone 18</label>
+				<description>The Controls for Zone 18</description>
+			</channel-group>
+			<channel-group id="source1" typeId="source1">
+				<label>Source 1</label>
+				<description>The Display Information for Source 1</description>
+			</channel-group>
+			<channel-group id="source2" typeId="source2">
+				<label>Source 2</label>
+				<description>The Display Information for Source 2</description>
+			</channel-group>
+			<channel-group id="source3" typeId="source3">
+				<label>Source 3</label>
+				<description>The Display Information for Source 3</description>
+			</channel-group>
+			<channel-group id="source4" typeId="source4">
+				<label>Source 4</label>
+				<description>The Display Information for Source 4</description>
+			</channel-group>
+			<channel-group id="source5" typeId="source5">
+				<label>Source 5</label>
+				<description>The Display Information for Source 5</description>
+			</channel-group>
+			<channel-group id="source6" typeId="source6">
+				<label>Source 6</label>
+				<description>The Display Information for Source 6</description>
+			</channel-group>
+		</channel-groups>
+
+		<config-description>
+			<parameter name="serialPort" type="text" required="false">
+				<context>serial-port</context>
+				<label>Serial Port</label>
+				<description>Serial Port to Use for Connecting to the Nuvo Amplifier</description>
+			</parameter>
+			<parameter name="host" type="text" required="false">
+				<context>network-address</context>
+				<label>Address</label>
+				<description>Host Name or IP Address of the Machine Connected to the Nuvo Amplifier (Serial over IP)</description>
+			</parameter>
+			<parameter name="port" type="integer" min="1" max="65535" required="false">
+				<label>Port</label>
+				<description>Communication Port (serial over IP). For IP Connection to the Nuvo Amplifier</description>
+				<default>4444</default>
+			</parameter>
+			<parameter name="numZones" type="integer" min="1" max="20" required="true">
+				<label>Number of Zones</label>
+				<description>Number of Zones On the Amplifier to Utilize in the Binding (Up to 20 Zones When Using Expansion Module)</description>
+				<default>6</default>
+			</parameter>
+			<parameter name="clockSync" type="boolean" required="false">
+				<label>Sync Clock On GConcerto</label>
+				<description>If Set to true, the Binding Will Sync the Internal Clock On the Grand Concerto to Match the openHAB
+					Host's
+					System Clock. The Sync Job Runs at Binding Startup and Once an Hour Thereafter. The Essentia G Has No RTC, So
+					This
+					Setting Has No Effect On That Component.</description>
```

Comment:
This formatting is a bit strange.

Reply:
Spotless did that :(

Reply:
I tried to improve it and committed the change but spotless is still not happy about it.

Reply:
Did you try removing all line breaks in the description tag and then apply spotless?

Reply:
It seems to have sorted itself out with the last change. No more spotless errors on this end.

Reply:
Spotless is quiet now, but the formatting looks still garbled.

Reply:
I am not sure why. It looks OK in the latest commit: https://github.com/openhab/openhab-addons/pull/7651/commits/a805ed70cb69620f43aeff74e6538eabcbcaa1f3#diff-4a125a54665ae3eaa126a6939adbbf36

Reply:
Now, it looks fine!